src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -308,7 +317,14 @@ public long getPreviousFlushTimeStampMs() {
   }
 
   public String getChannelName() {
-    return this.channel.getFullyQualifiedTableName();
+    return this.channel.getFullyQualifiedName();
+  }
+
+  public String toString() {
+    return "StreamingChannelName:"
+        + getChannelName()
+        + ", FullyQualifiedTableName:{}"
+        + this.channel.getFullyQualifiedTableName();
```

Comment:
why not just use this.channel.getFullyQualifiedName? it will contains both channel name and table name

Reply:
Sounds like an idea.. let me do that..